model/src/main/resources/reference.conf

```diff
@@ -117,6 +117,7 @@ ort {
 
         deltaScans = "true"
         deltaScanLimit = "10"
+        timeout = "3600000"
```

Comment:
As the timeout is a rather big value (in the range of minutes up to an hour), I would suggest using a different time unit than milliseconds. Maybe minutes would be more appropriate. WDYT?

Reply:
Yes this is a good idea. I would go for minutes too.